package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "test:deps": "dependency-check . && dependency-check . --extra --no-dev -i es2040",
     "test:lint": "standard",
-    "test:node": "NODE_ENV=test bin/test",
+    "test:node": "NODE_ENV=test tape **/*.test.js",
```

Comment:
ooh damn that's a nice trick. I didn't notice.
Happy to change that back. 

Just by the way: I have a vim binding that will send the current buffer to node. So if I press the key '1' when I'm looking at a test file then I see the result of running those tests in my vim window.